### PR TITLE
Repair console state and readiness flows

### PR DESCRIPTION
**Pull request by flybaozi**
## Summary

- Keep Console readiness current after setup, require one usable Signal Routing Rule owned by an active Agent, and remove the superseded binding API and duplicate route Agent field.
- Preserve AIGateway credential health across active-provider enable calls, reset it only on real operator transitions, and publish the complete validation contract.
- Harden Console loading, filtering, not-found, draft, pending-save, IME, encrypted-setting, and Agent-scope behavior across the affected pages.
- Keep Automation Job detail on its owning-Agent route and permission while making generated links carry that owner explicitly.
- Keep unbounded Cron schedules continuous, make completed finite schedules read-only, and preserve occurrence bounds when editing timing.
- Localize the affected Setup and Console feedback in all four catalogs and stop periodic plugin reconciliation from racing SQL Sandbox tests.

## Root cause

The large UI pass mixed server-owned state with fresh list caches, placeholder rows, and dialog-local state. Several follow-up fixes then widened the problem by resetting current runtime health or preserving stale data across an Agent scope change. This change returns each decision to its owning API or domain state and removes the superseded paths.

## Validation

- Webapps: 167 tests passed; TypeScript type-check, production Vite build, and formatting passed.
- Control Plane: 2053 tests passed with 9 excluded; warnings-as-errors compile and formatting passed.
- Locale catalog tests passed in all four languages.
- The OpenAPI document and generated client are current; `git diff --check` passed.
- Real Chrome acceptance on a temporary database clone verified Permission Grant 404/error handling, Background action reset across Agent changes, Brain selection reset across owner changes, and Skill Experience draft reset across scope changes. No business mutation was submitted, and the temporary database was deleted.

## Summary by CodeRabbit

- **New Features**
  - Re-enable disabled AI providers from the Console.
  - Improved deployment readiness checks, including signal-route status.
  - Added unsaved-change protection, discard confirmations, read-only forms, loading skeletons, and richer editor controls.
  - Enhanced filtering and navigation across agents, conversations, jobs, schedules, Brain tools, and background jobs.
  - Added schedule run, pause, resume, and delete controls.
- **Bug Fixes**
  - Improved provider health, resource scoping, job ownership, status displays, stale-data handling, uploads, and worker load visibility.
- **Localization**
  - Expanded translations across English, Japanese, Korean, and Simplified Chinese.

### CODE REVIEW

File: app/control_plane/lib/ankole/ai_gateway/provider_configs.ex [hunk 1]
```diff
@@ -149,27 +149,20 @@ defmodule Ankole.AIGateway.ProviderConfigs do
...
-        else
-          nil -> {:error, :not_found}
-          {:error, _reason} = error -> error
+    Repo.transact(fn repo ->
+      with %Provider{} = provider <- lock_provider(repo, provider_id),
+           {:ok, attrs} <- provider_attrs_for_write(attrs, provider),
+           {:ok, provider} <- provider |> Provider.changeset(attrs) |> repo.update() do
+        if credential_pool_changed? do
+          clear_pool_health(provider)
         end
```
[the remark this answers is not in the document]
Comment (flybaozi) [reply app/control_plane/lib/ankole/ai_gateway/provider_configs.ex]: Fixed in 8c96f983. The pool now persists an internal `health_revision` for each credential. Operator replacement and re-enabling a disabled provider rotate it; metadata-only updates and automatic OAuth refresh preserve it. Health, rate-limit state, and affinity are keyed by `{credential_id, health_revision}`, and runtime outcomes carry the selected stored entry. This removes transaction-side health resets, so an old snapshot or a rolled-back write cannot affect the committed replacement. Regression tests cover an already-dead old revision, a usable new revision, late old failure and success, new affinity, and active versus disabled enable behavior. The full control-plane suite passes: 2055 tests, with 9 excluded.

File: app/locales/zh-Hans-CN.toml [hunk 2]
```diff
@@ -1002,19 +1009,15 @@ brain_embedding_error_model_agent = "请选择一个已经配置 embedding Model
...
 brain_embedding_open_agents = "前往配置 Agent"
 brain_embedding_unavailable_agent = "当前配置的 Agent {$uid} 不再 active 或已没有 embedding ModelProfile；保存前请选择其他 Agent。"
-discard = "放弃修改"
-discard_description = "尚未保存的配置修改将会丢失。"
-discard_title = "放弃未保存的修改？"
-editor_description = "以 JSON 形式编辑该配置值。"
+editor_description = "编辑该配置值。"
 empty_title = "暂无设置"
-encrypted_value_hint = "保留掩码会保留当前密文；可输入新的 JSON 值替换，或点击右侧按钮查看。"
+encrypted_value_hint = "保留掩码会保留当前密文；可输入原样密钥，结构数据则输入有效的 JSON 对象或数组。"
```
[the remark this answers is not in the document]
Comment (flybaozi) [reply app/locales/zh-Hans-CN.toml]: Fixed in 8c96f983. The Simplified Chinese hint now says `原始字符串` and keeps the existing masked-value and JSON object or array guidance.

File: app/webapps/console/pages/brain-status.tsx [hunk 3]
```diff
@@ -57,9 +57,15 @@ export function BrainStatusPage() {
...
-          </StatusIndicator>
+          {/* The warning tone is reserved for a report that arrived degraded; a
+              pending or failed query shows a neutral placeholder or nothing. */}
+          {status ? (
+            <StatusIndicator tone={status.status === 'ok' ? 'positive' : 'warning'}>
+              {textValue(status.status) ?? '—'}
+            </StatusIndicator>
+          ) : principals.isPending || (Boolean(ownerUID) && query.isPending) ? (
+            <Skeleton className="h-6 w-24" />
+          ) : null}
```
[the remark this answers is not in the document]
Comment (flybaozi) [reply app/webapps/console/pages/brain-status.tsx:68]: Verified and left unchanged. `Ankole.Brain.Status.run/1` returns a complete report with `status` equal to `ok` or `error` when the query succeeds; a producer failure leaves query data absent. An empty `memory_status` object is outside that producer and schema contract. Changing only the badge would also leave the lower cards rendering malformed zero values, so this partial defensive check would hide a contract violation instead of handling it correctly.

File: app/webapps/console/pages/conversations.tsx [hunk 4]
```diff
@@ -119,11 +117,12 @@ export function ConversationsListPage() {
...
   }
 
-  const isFiltered = Boolean(searchFilter.trim()) || activeFilter !== null
+  const isFiltered = Boolean(searchFilter.trim()) || activeFilter !== null || Boolean(scope.agentUID)
   const clearFilters = () => {
     const next = new URLSearchParams(searchParams)
     next.delete('q')
     next.delete('active')
+    next.delete('agent')
     setSearchParams(resetCursorParams(next), { replace: true })
```
[the remark this answers is not in the document]
Comment (flybaozi) [reply app/webapps/console/pages/conversations.tsx:126]: Verified and left unchanged. `min_messages=0` is an opt-in visibility mode that broadens the list by removing the default `min_messages=2`; it is not a narrowing filter. If `showAll` alone produces an empty list, no conversations exist, so a filter-empty message and clear action would be incorrect. When clearing `q`, `active`, or `agent`, preserving `showAll` also maximizes the visible results. This behavior predates this branch and matches the page copy.

File: app/webapps/console/pages/workers.tsx [hunk 5]
```diff
@@ -646,11 +663,11 @@ function UploadDialog({
...
   }
 
+  // Each file reports its own outcome through its own toast, so the dialog
+  // hands them over and closes rather than tracking a batch result.
   const submit = () => {
     if (disabled || files.length === 0) return
-    for (const file of files) {
-      onUpload({ root, file }, joinPath(currentPath, subdir.trim(), file.name))
-    }
+    for (const file of files) onUpload({ file, workerPath: joinPath(currentPath, subdir.trim(), file.name) })
```
[the remark this answers is not in the document]
Comment (flybaozi) [reply app/webapps/console/pages/workers.tsx:670]: Verified, but no change belongs in this PR. Leading-slash normalization is owned by the control-plane `FileTransferLane`, not by the reviewed `workers.tsx` hunk. The normalized path stays inside the selected virtual root, and Agent Computer also applies a realpath boundary check, so this is not a path escape or security issue. It is a pre-existing relative-input contract mismatch that needs a separate boundary change with lane and controller tests; a frontend-only rejection would duplicate policy and leave other callers unchanged.

File: app/webapps/setup/app.tsx [hunk 6]
```diff
@@ -365,13 +377,14 @@ function IdentityStep({
...
   useSignals()
+  const { t } = useTranslation()
   const query = useQuery({
     queryKey: ['setup-identity-provider-adapters'],
     queryFn: () => internalAPIGet<{ adapters: IdentityAdapter[] }>('/.internal-apis/setup/identity-provider-adapters')
   })
   const adapters = filterSelectedPluginItems(query.data?.adapters ?? [], pluginsModel.selectedPluginIDs.value)
 
-  if (query.isLoading) return <Panel title="">{i18n.t('common.loading')}</Panel>
+  if (query.isLoading) return <Panel title={t('common.loading')}>{t('common.loading')}</Panel>
```
[the remark this answers is not in the document]
Comment (flybaozi) [reply app/webapps/setup/app.tsx]: Fixed in 8c96f983. The loading panel now uses `setup.identity_provider` as its title and keeps `common.loading` only in the body.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,12 @@
...
+- Make Console state match visible resources. Agent and Brain owner changes remove prior-scope rows from schedule, Signal Routing Rule, webhook, conversation, Automation Job, Background Agent Job, and Brain lists. Background Agent Job action dialogs close with their Agent filter, while Automation Job details retain owning-Agent authorization. Missing details show "page not found"; dirty drafts require confirmation; pending writes cannot be dismissed as discarded; Skill Experience drafts reset per Agent and retain their conflict hash; schedule limits survive edits; terminal schedules are read-only; and IME input cannot submit forms. Provider-hosted capability switches can be turned off again. Setup validates activation codes in the selected language and shows submission progress. Searches, settings, and operational states now use localized visible labels and explicit errors.
+- Keep startup Signal connection reconciliation in tests, but disable its periodic timer so supervised plugin work does not race SQL Sandbox tests.
+
```
[the remark this answers is not in the document]
Comment (flybaozi) [reply CHANGELOG.md]: Updated in 8c96f983. Version 0.70.0 now describes the observable outcomes for revision-isolated provider health, encrypted scalar versus structured JSON settings, compact Background Agent Job actions, Automation Job positive IDs and explicit errors, and the retained test reconciliation behavior. I did not add file or test inventories because this repository rule requires user and operator outcomes and explicitly rejects implementation inventories. The four bullets cover the material retained changes.
